pkg/k8shandler/collection.go

```diff
@@ -261,6 +261,11 @@ func isDaemonsetDifferent(current *apps.DaemonSet, desired *apps.DaemonSet) (*ap
 		different = true
 	}
 
+	if utils.AreResourcesDifferent(current, desired) {
+		logrus.Infof("Daemonset resource(s) change found, updating %q", current.Name)
```

Comment:
Use `Collector` instead of `Daemonset` to be consistent with the other log messages below

Reply:
updated